### PR TITLE
Add new facilitator survey to post-email for CSD/CSP

### PR DESCRIPTION
**Pull request by molly-moen**
Update our Facilitator post-workshop auto-email to send the new Foorm survey for any non-summer CSD/CSP workshop (we don't send the email for CSF). The survey will be populated with workshop course, workshop subject, and workshop id. This PR also adds the ability to send any variable to a Foorm misc survey via a url parameter.
 

### Future work
After 9/5/2020 we will want to deprecate the old JotForm version of the survey, but we have one more summer workshop before then that we want to have survey data in the same bucket as the other workshops this summer.

## Links

- [jira](https://codedotorg.atlassian.net/browse/PLC-962)

## Testing story
Validated via the mailer previews that an academic year workshop will show the new survey with the correct variables and a summer workshop will show the old survey.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/mailers/pd/workshop_mailer.rb [hunk 1]
```diff
@@ -154,7 +154,14 @@ def facilitator_pre_workshop(user, workshop)
   def facilitator_post_workshop(user, workshop)
     @user = user
     @workshop = workshop
-    @survey_url = CDO.studio_url "/pd/misc_survey/facilitator_post", CDO.default_scheme
+    # TODO: After 9/5/2020 move all workshops to the new survey (9/5 is last 2020 Summer Workshop)
```
Comment (clareconstantine) [inline dashboard/app/mailers/pd/workshop_mailer.rb:157]: Would you mind making a Jira item for this, if we don't already have one?

**Review by clareconstantine — APPROVED** [review]
Looks good! Also, seems super useful to be able to pass in params via the url!

File: dashboard/app/mailers/pd/workshop_mailer.rb (hunk 1 above)
Comment (molly-moen) [reply dashboard/app/mailers/pd/workshop_mailer.rb:157]: will do!

Comment (molly-moen) [reply dashboard/app/mailers/pd/workshop_mailer.rb:157]: Jira item is [here](https://codedotorg.atlassian.net/browse/PLC-970) we will want to circle back with MeganH when the time so she knows it's done.

**Pull request merged**